go/v1beta1/main/main.go

```diff
@@ -14,12 +16,62 @@ func main() {
 		log.Panicf("Error when registering storage type providers, %s", err)
 	}
 
-	// register a new storage type using the key 'mongodb'
-	err = grafeasStorage.RegisterStorageTypeProvider("mongodb", storage.ElasticsearchStorageTypeProvider)
+	logger, err := createLogger(true)
+	if err != nil {
+		log.Fatalf("failed to create logger: %v", err)
+	}
+
+	esClient, err := createESClient(logger)
+	if err != nil {
+		logger.Fatal("failed to connect to Elasticsearch", zap.NamedError("error", err))
+	}
+
+	elasticsearchStorage := storage.NewElasticsearchStore(esClient, logger.Named("Elasticsearch Store"))
 
+	// register a new storage type using the key 'elasticsearch'
+	err = grafeasStorage.RegisterStorageTypeProvider("elasticsearch", elasticsearchStorage.ElasticsearchStorageTypeProvider)
 	if err != nil {
-		log.Panicf("Error when registering my new storage, %s", err)
+		logger.Fatal("Error when registering my new storage", zap.NamedError("error", err))
+	}
+
+	err = server.StartGrafeas()
+	if err != nil {
+		logger.Fatal("Failed to start Grafeas server...", zap.NamedError("error", err))
+	}
+}
+
+func createESClient(logger *zap.Logger) (elasticsearch.Client, error) {
+	c, err := elasticsearch.NewClient(elasticsearch.Config{
+		Addresses: []string{
+			"http://db:9200",
```

Comment:
Left these hardcoded for now until we decide on whether or not to restructure this project or instantiate Grafeas in another way.